src/07_slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1])
 
 # Output the second-to-last element: 9
-print()
+print(a[len(a)-2])
```

Comment:
I like that you made these dynamic by passing in the length of the list, good job!